src/components/bark/login/bark-login-form.tsx

```diff
@@ -111,12 +111,12 @@ export default function BarkLoginForm(props: {
           </BarkFormParagraph>
         )}
         <BarkFormInput form={form} name="otp" label="Enter OTP" />
-        <BarkFormSubmitButton>Login</BarkFormSubmitButton>{" "}
-        <Link href={RoutePath.ROOT}>
-          <Button variant="secondary" type="button">
+        <div className="mt-6 flex w-full gap-x-4">
+          <Button variant="brandInverse" type="button" className="">
             Cancel
           </Button>
```

Comment:
The Link to RoutePath.ROOT appears to be missing.

Reply:
Done c0078744b6f86a7eed3dd82655952a918f1b9f31